caom2-artifact-resolvers/build.gradle

```diff
@@ -14,7 +14,7 @@ sourceCompatibility = 1.8
 
 group = 'org.opencadc'
 
-version = '1.2.2'
+version = '1.3'
```

Comment:
version 1.2.3 is fine - yes it's new functionality but it is binary compatible and don't want that large increment. Also, when that does happen it should be 1.3.0

Reply:
I thought we want to mark the use of the new storage with a major version. Changed to `1.2.3`